editor/src/components/inspector/resize-to-fit-control.spec.browser2.tsx

```diff
@@ -22,6 +26,21 @@ describe('Resize to fit control', () => {
     expect(view.style.minHeight).toEqual('')
     expect(view.style.maxHeight).toEqual('')
   })
+
+  it('resizes to fit, with shortcut', async () => {
+    const editor = await renderTestEditorWithCode(project, 'await-first-dom-report')
+    const view = await selectView(editor)
+    await expectSingleUndoStep(editor, async () =>
+      pressKey('r', { modifiers: { alt: true, cmd: true, shift: true, ctrl: false } }),
+    )
+
+    expect(view.style.width).toEqual(MaxContent)
+    expect(view.style.minWidth).toEqual('')
+    expect(view.style.maxWidth).toEqual('')
+    expect(view.style.height).toEqual(MaxContent)
+    expect(view.style.minHeight).toEqual('')
+    expect(view.style.maxHeight).toEqual('')
```

Comment:
should this check if it removes flex-grow, flex-shrink, flex too?

Reply:
great suggestion! fixed in [9680425](https://github.com/concrete-utopia/utopia/pull/3197/commits/9680425700d693a3cfba8134e355f4363e3a0eac)